spec/controllers/account/tenants_controller_spec.rb

```diff
@@ -1,22 +1,41 @@
 require 'rails_helper'
+require 'support/render_views'
```

Comment:
without this line it doesn't work?
in rails_helper include it if not included
with code something like
Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }

check if not added yet

Reply:
it's commented in `spec/rails_helper.rb`
just uncomment that line and remove this `require`